### PR TITLE
871738: Added UG documentation for new TextArea component

### PR DESCRIPTION
**Pull request by arunkumarssf3997**
### Bug description
Added UG documentation for new TextArea component.

### Root cause
NA

### Solution description
Added UG documentation for new TextArea component.

### Reason for not identifying earlier
 * [ ] Guidelines not followed. If yes, provide which guideline is not followed.

 * [ ] Guidelines not given. If yes, provide which/who need to address.
    Tag label `update-guideline-coreteam` or `update-guideline-productteam`. 

 * [x] If any other reason, provide the details here. 
     
### Areas tested against this fix
Provide details about the areas or combinations that have been tested against this code changes.
* [ ]  Tested against feature matrix. [Feature matrix link](https://syncfusion.sharepoint.com/sites/EJ2ProductOwners/Shared%20Documents/Forms/AllItems.aspx?viewid=ae81c682%2D3d0f%2D462a%2Db8ec%2D7358748d489d&id=%2Fsites%2FEJ2ProductOwners%2FShared%20Documents%2FGeneral%2FFeature%20Matrix%20%2D%20Documents)
* [x]  NA

### Is it a breaking issue?
* [ ]  Yes, Tag `breaking-issue`. 
* [x]  NO
 
 If yes, provide the breaking commit details / MR here. 
 
### Action taken
NA

 Feature matrix document updated
* [ ]  Yes
* [ ]  NO
* [x]  NA
 
Automation details - Mark `Is Automated` field as (Yes, Manual, Not Applicable) in corresponding JIRA task once the bug is automated. 
* [ ] BUnit, share corresponding MR.
* [ ] E2E or Manual Automation using tester - Make sure all items are automated with priority before release which can be tracked in [automation dashboard](https://syncfusion.atlassian.net/secure/Dashboard.jspa?selectPageId=43396).
 
If the same issue is reproduced in ej2, what will you do?
* [ ]  Resolved. Provide MR link.
* [ ]  NO. Created task to track it. Share task link. 
* [x]  NA
 
 Is this common issue need to be addressed in the same component or on other components in our platform? 
* [ ]  Yes - Need to check in other components, tag `needs-attention-coreteam` 
* [x]  No
  
### Output screenshots
NA

### Blazor Checklist
Confirm whether this feature is ensured in both Blazor Server and WASM
* [x]  NA
* [ ]  Yes
* [ ]  NO

Is there any new API or existing API name change?
* [ ]  Yes. If yes, Provide API Review task link.
* [x]  NO
  
Is there any existing behavior change due to this code change?
* [ ]  Yes. Add `breaking-change` label.
* [x]  NO

Do the code changes cause any memory leak and performance issue? (Test only if you suspect that your code may cause problem)
* [ ]  Yes
* [x]  NO

### Reviewer Checklist
* [x]  All provided information are reviewed and ensured.

### CODE REVIEW

File: ej2-javascript/textarea/rows-columns.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+platform: ej2-javascript
+control: Rows and Columns Count
+publishingplatform: ##Platform_Name##
+documentation: ug
+domainurl: ##DomainURL##
+---
+
+# Rows and Columns in ##Platform_Name## TextArea control
+
+Two essential attributes, "rows" and "columns," play a pivotal role in customizing the TextArea's appearance and layout.
```
Comment (saravanangn) [inline ej2-javascript/textarea/rows-columns.md:14]: use symbol to highlight the words like `rows` and `columns`

File: ej2-javascript/textarea/resize.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+platform: ej2-javascript
+control: Resize
+publishingplatform: ##Platform_Name##
+documentation: ug
+domainurl: ##DomainURL##
+---
+
+# Resize in ##Platform_Name## TextArea control
+
+TextArea can be resized to adjust their size according to specific requirements. This resizing behavior can be enabled and configured using the [`resizeMode`](../api/textarea/#resizeMode) API, which offers several options for resizing the TextArea:
```
Comment (saravanangn) [inline ej2-javascript/textarea/resize.md:14]: The TextArea allows users to input and edit large amounts of text. Resizing this control effectively can enhance the user experience and accommodate varying content needs. This resizing behavior can be enabled and configured using the [resizeMode](../api/textarea/#resizeMode) API, which offers several options for resizing the TextArea.

File: ej2-javascript/textarea/resize.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+| Type  | Description |
+| -- | -- |
+| Vertical  | Allows users to adjust the height of the TextArea vertically. It is suitable when you want to resize the TextArea only along the vertical axis, accommodating varying amounts of text input. |
```
Comment (saravanangn) [inline ej2-javascript/textarea/resize.md:18]: Allows users to adjust the height of the TextArea vertically. It is suitable when users want to resize the TextArea only along the vertical axis, accommodating varying amounts of text input.

File: ej2-javascript/textarea/resize.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+| Horizontal | Allows users to adjust the width of the TextArea horizontally. This option is beneficial when developers need to accommodate longer lines of text within the TextArea while maintaining a fixed height. |
```
Comment (saravanangn) [inline ej2-javascript/textarea/resize.md:19]: Users can adjust the width of the TextArea horizontally. This option is helpful for accommodating longer lines of text without altering the height of the control.

File: ej2-javascript/textarea/resize.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+| Both | Enables users to resize both the height and width of the TextArea, providing maximum flexibility in adjusting its size. This option is suitable for scenarios where users require complete control over the dimensions of the TextArea. |
```
Comment (saravanangn) [inline ej2-javascript/textarea/resize.md:20]: Allows users to adjust both the height and width of the TextArea, offering maximum flexibility in resizing. It is ideal for situations where users need precise control over the dimensions of the TextArea.

File: ej2-javascript/textarea/resize.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+| None | Disables the resize in TextArea. This option is ideal for situations where you want to maintain a fixed size for the TextArea, preventing any unintended changes by users. |
```
Comment (saravanangn) [inline ej2-javascript/textarea/resize.md:21]: Disables the resizing in the TextArea. This option is ideal for situations where maintaining a consistent layout is critical, and resizing by users is unnecessary.

**Review by PrinceOliver — APPROVED** [review]

**Pull request merged**